modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.js

```diff
@@ -37,3 +63,50 @@ const getSiteURL = () => {
 
 	return '';
 };
+
+const getProducts = async () => {
+	const products = fetcher(
+		`/o/headless-commerce-admin-catalog/v1.0/products`
+	);
+
+	return products;
+};
+
+const getUrlString = (url) => {
+	const urlSplit = url.split('/');
+	const filteredUrl = urlSplit[urlSplit.length - 1];
+
+	return filteredUrl;
+};
+
+const setCategoryLabels = async () => {
+	const currentUrl = Liferay.currentURL;
+	const labelDiv = document.getElementById('category-label');
```

Comment:
Please, since the element is inside the fragment scope you can use fragmentElement instead of the document because the document is used when I have to access external elements from the fragment.

Reply:
Done!